diff --git a/Makefile b/Makefile
index 87b35e2..80f13d9 100644
--- a/Makefile
+++ b/Makefile
@@ -52,12 +52,6 @@ elf: $(ELF)
 
 asm: $(ASM)
 
-lib/version.c: .git/HEAD .git/index
-	echo "const char *gitid = \"$(shell git describe --always --dirty)\";" > lib/version.c
-	echo "const char *gitdate = \"$(shell git log -n 1 --date=short --format=format:"%ad.%h" HEAD)\";" >> lib/version.c
-	echo "const char *gitversion = \"$(shell git rev-parse HEAD)\";" >> lib/version.c
-#	echo "const char *gitstatus = \"$(shell git status -s )\";" >> lib/version.c
-
 zsbl/ux00boot.o: ux00boot/ux00boot.c
 	$(CC) $(CFLAGS) -DUX00BOOT_BOOT_STAGE=0 -c -o $@ $^
 
